server/tests/runs.e2e-spec.ts

```diff
@@ -0,0 +1,341 @@
+            data: {
+                steamID: '123654789',
+                country: 'CA',
+                alias: 'John Laurie',
+                roles: Roles.MAPPER,
+                bans: 0,
+                avatar: 'jlau.png',
+                profile: {
+                    create: {
+                        bio: 'Canadian educator and political activist. He espoused Indigenous causes and assisted many Indigenous youth in furthering their education.'
+                    }
+                }
+            },
+            include: {
+                profile: true
+            }
+        });
+
+        map1 = await prisma.map.create({
+            data: {
+                name: 'rj_super_jurf_testing',
+                type: MapType.RJ,
+                statusFlag: MapStatus.APPROVED,
+                submitter: { connect: { id: user1.id } },
+                credits: {
+                    create: {
+                        type: MapCreditType.AUTHOR,
+                        user: { connect: { id: user1.id } }
+                    }
+                },
+                info: {
+                    create: {
+                        description: "This breaking bad themed jurf map is great for fans of the show and fast-paced rocket action alike.",
+                        numTracks: 20,
+                        creationDate: new Date()
+                    }
+                }
+            }
+        });
+
+        map2 = await prisma.map.create({
+            data: {
+                name: 'surf_epicfun_testingmap',
+                type: MapType.SURF,
+                statusFlag: MapStatus.APPROVED,
+                submitter: { connect: { id: user2.id } },
+                credits: {
+                    create: {
+                        type: MapCreditType.AUTHOR,
+                        user: { connect: { id: user2.id } }
+                    }
+                },
+                info: {
+                    create: {
+                        description: "Momentum Mod is not responsible for any bodily harm that may occur as a result of playing this map.",
+                        numTracks: 2,
+                        creationDate: new Date()
+                    }
+                }
+            }
+        });
+
+        run1 = await prisma.run.create({
+            data: {
+                map: { connect: { id: map1.id } },
+                player: { connect: { id: user1.id } },
+                trackNum: 1,
+                zoneNum: 1,
+                ticks: 113700,
+                tickRate: 66.6,
+                flags: 1 << 0,
+                file: 'world_record.run',
+                hash: '0xdeadbeef',
+                rank: {
+                    create: {
+                        gameType: MapType.RJ,
+                        flags: 0,
+                        trackNum: 0,
+                        zoneNum: 0,
+                        rank: 6,
+                        createdAt: new Date(),
```

Comment:
Why the `createdAt `here? This is handled by Prisma (note `creationDate` on `Map`s is an actual date we track ourselves, since that could be for some map originally released years ago) . Same with other `Run` creations in here.